app/models/spree/digital_link.rb

```diff
@@ -22,6 +22,10 @@ def ready?
       attachment.exists?
     end
 
+    def is_file?
```

Comment:
I know you took the naming from the controller, but I think it is (and was already before) a really bad name. I think a much better name for this would be `attachment_exists?` or `file_exists?`.